libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.m

```diff
@@ -169,4 +196,21 @@ - (DDLogLevel)readLogLevel {
     return logLevel;
 }
 
+static inline DDLogFlag DDLogFlagForLogLevel(DDLogLevel level) {
+    switch (level) {
+        case DDLogLevelOff:
```

Comment:
You're right - it should probably be 0 for correctness. Looking at the implementation, it looks like flags are for fine-grained logging or for custom levels by using bitmask directly. Since the operation is bitwise `&` and the `level` is 0 for `OFF`, wouldn't the end result be 0 anyway no matter what the flag is? `DDLogFlag` is meant for fine-tuning log levels (which we don't really use in this case). See doc [here](http://cocoadocs.org/docsets/CocoaLumberjack/2.0.0/Constants/DDLogFlag.html). I guess the flag value should also be `DDLogLevelOff` in this case.

Good find on the `Cocoapod` doc - I was going crazy reading the doc within `Xcode` and looking at code to figure out the inner workings.